### ISSUE TITLE
No validation for presence of `start_date` and `end_date` in competition form

### ISSUE THREAD

**Issue by dunkOnIT** [thewca/worldcubeassociation.org#9277]
This leads to errors if you try to save the form without filling them in

**Comment by kr-matthews** [comment]
Isn't this a backend issue? The frontend is just reading the list of errors that it receives.

**Comment by dunkOnIT** [comment]
Good point! Fix pending at #12685 - although there was still the issue that the 500 error that got returned by the backend wasn't handled in the frontend, so I'm changing the issue description.
